### PR TITLE
Automatically port Reflection.PortableExecutable.DebugDirectory* APIs

### PR DESCRIPTION
**Pull request by carlossanlop**

### CODE REVIEW

**Comment by BillWagner** [comment]
closing and reopening for a clean build.

File: xml/System.Reflection.PortableExecutable/DebugDirectoryEntryType.xml [hunk 1]
```diff
@@ -42,7 +42,16 @@
       </ReturnValue>
```
Comment (rpetrusha) [inline xml/System.Reflection.PortableExecutable/DebugDirectoryEntryType.xml]: It would be good to add an enum type description.

**Comment by carlossanlop** [comment]
@tmat your help would be hugely appreciated here as well. There are a few extra fields that need a summary (they were not documented in source).

File: xml/System.Reflection.PortableExecutable/DebugDirectoryEntryType.xml [hunk 2]
```diff
@@ -134,7 +157,17 @@
...
       <MemberValue>16</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The presence of this entry indicates a deterministic PE/COFF file. See the Remarks section for more information.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
```
Comment (tmat) [inline xml/System.Reflection.PortableExecutable/DebugDirectoryEntryType.xml]: ># Remarks [](start = 1, length = 9)

Empty remarks?

File: xml/System.Reflection.PortableExecutable/DebugDirectoryEntryType.xml [hunk 3]
```diff
@@ -158,7 +191,7 @@
       </ReturnValue>
       <MemberValue>0</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>An unknown value that is ignored by all tools.</summary>
```
Comment (tmat) [inline xml/System.Reflection.PortableExecutable/DebugDirectoryEntryType.xml]: "should be ignored"?

**Pull request merged**